src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java

```diff
@@ -239,6 +239,10 @@ else if(key.equals("inSubset")) {
             }
 
         });
+        
+        if (EntitySearcher.getAnnotationObjects(node, ontology).count() > 0 && term.getCurie() == null) {
```

Comment:
I would swap the logic on this put the `term.getCurie() == null` first that way it will only check the other condition when terms don't have a curie. (Maybe more efficient)

Reply:
done